src/View/StringTemplate.php

```diff
@@ -369,4 +369,46 @@ public function addClass($input, $newClass, $useIndex = 'class')
 
         return $input;
     }
+
+    /**
+     * Removes a class and returns a unique list either in array or space separated
+     *
+     * @param array|string $input The array or string to remove the class from
+     * @param array|string $removeClass the new class or classes to remove
+     * @param string $useIndex if you are inputting an array with an element other than default of 'class'.
+     * @return array|string
```

Comment:
Same here probably.

Reply:
These would be the same with `addClass()`

When I get back to this, should I change those to:

`* @param array string[]|string`

or

`* @param string[]|string`

Or is there a particular format?

Reply:
`@param string[]|string`

Reply:
This is updated, and I'll work on getting tests up and running ASAP